platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/annotations/Hole.java

```diff
@@ -12,7 +12,7 @@
 /**
  * Encapsulates a {@link List} of {@link LatLng} points defining a hole
  */
-public class Hole extends ArrayList<LatLng> implements Parcelable {
+class Hole extends ArrayList<LatLng> implements Parcelable {
```

Comment:
Didn't find a way to create a `Parcelable` of `List<List<LatLng>>`.
Do you know how?